### DISCUSSION TITLE
qq VIP帐号购买的两年期，一直收不到邮件验证码

### DISCUSSION THREAD

**Discussion by ymhhh** [getlantern/lantern-forum-cn#700] [General]
帐号：ymhhh@vip.qq.com
![Uploading WechatIMG14.jpg…]()

**Comment by ymhhh** [comment]
![WechatIMG14](https://github.com/getlantern/lantern-forum-cn/assets/11403264/92fa1d30-3a45-4301-9113-4c5f3c6ede4c)

**Comment by MaxMKMax** [comment]
@ymhhh 您好. 我们会帮您解决问题. 请您放心. 为了保护您的隐私, 请您直接在这个链接上提交您的资料就好了. 支付单据和邮箱. 
后续您可以在GitHub这个帖子和我跟进如果您的问题还没解决. 谢谢您!
https://lantern.surveysparrow.com/s/propayment-triage-form/tt-ZxBeJ

*备注: 如果各位看到这个链接, 请您依然开个帖子说明您的问题. 这样我们才能发现到您的问题帮您跟进. 不要只点连接提交问题而不开帖子. 帖子上不需要个人资料了. 邮箱和支付单据放在链接里就好了. 谢谢!

**Reply by ymhhh** [reply]
你好，链接已经失效了。

**Reply by MaxMKMax** [reply]
@ymhhh 您好. 请您放心, 我已经帮您手动提交问题了. 等待回复. 有消息马上通知您. 过几天您也查看邮箱或这个帖子.

**Reply by ymhhh** [reply]
好的 ，谢谢

**Reply by wxl0618** [reply]
> @ymhhh 您好. 请您放心, 我已经帮您手动提交问题了. 等待回复. 有消息马上通知您. 过几天您也查看邮箱或这个帖子.

我的蓝灯VIP帐号购买的两年期，最近一直收不到邮件验证码，怎么回事啊 我的邮箱是：wenxiaoling0618@hainmc.edu.cn

**Reply by MaxMKMax** [reply]
@wxl0618 
您好. 您可以提供我支付单据吗? 没收到是因为第三方系统被墙挡住了. 所以我们会手动处理.

**Comment by MaxMKMax** [comment]
@ymhhh 
您好. 这是您的激活码:
62YRK-7WBYD-6JX94-2BKWQ-9DCQT
